docs/DeveloperGuide.md

```diff
@@ -237,7 +237,72 @@ The following activity diagram summarizes what happens when a user executes a ne
   * Pros: Will use less memory (e.g. for `delete`, just save the person being deleted).
   * Cons: We must ensure that the implementation of each individual command are correct.
 
-_{more aspects and alternatives to be added}_
+_{more aspects and alternatives to be added}
+
+### Add a patient
+
+#### Implementation
+
+The add patient feature is facilitated by `AddCommand`, `AddCommandParser` and `Person`. 
+
+Given below is an example usage scenario and how the add patient feature behaves at each step.
+
+Step 1. The user launches the application for the first time.
+
+Step 2. The user executes an Add Command (e.g. 'add n\Alice ad\01/01/2022 dob\01/01/2002 ward\WA') to add a new patient to the address book. 
```

Comment:
the add command here has some errors
1. doesn't have the ic\ field
2. ward\ should just be w\